dist/s_executable

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+cd ../test/suite
+if [[ $(find . -executable -type f | grep -v -w "./run.py") ]]; then
+    echo "failed test"
```

Comment:
My suggestion here would be to echo nothing at all in the success case, and to print the filename (and problem) when failing. e.g. `echo "${file} incorrectly marked executable"`